src/linelast_solver.cpp

```diff
@@ -417,6 +418,28 @@ void LinElastSolver::SetupDomainBCOperators()
    }
 }
 
+void LinElastSolver::SetParameterizedProblem(ParameterizedProblem *problem)
+{
+   // Set lambda and mu coeffs
+   Vector lambda(numBdr);
+   Vector mu(numBdr);
+
+   function_factory::linelast_disp::fill_lambda(lambda);
+   function_factory::linelast_disp::fill_mu(mu);
+   
+   delete lambda_c;
+   lambda_c = new PWConstCoefficient(lambda);
```

Comment:
This is just checking: are we always setting `lambda` and `mu` to be piecewise constant for any problem?